nexus/types/src/deployment.rs

```diff
@@ -227,6 +227,12 @@ pub struct Blueprint {
     /// driving the system to the target release.
     pub target_release_minimum_generation: Generation,
 
+    /// The generation of the active group of Nexuses
+    ///
+    /// If a Nexus instance notices it has a nexus_generation less than
+    /// this value, it will start to quiesce (see: RFD 588).
```

Comment:
```suggestion
    /// this value, it will start to quiesce in preparation for handing off control to the newer generation (see: RFD 588).
```

Reply:
Updated in https://github.com/oxidecomputer/omicron/pull/8944/commits/eafafe4a8f6dfa4c5e7fa436e675f5956aa4363e